CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ the code was deployed.
 
 ## [Unreleased]
 
+### Changed
+- Updated favicon image from old logo to current logo
```

Comment:
Good!

Reply:
Not very important for this deployment, but we should include the clickup task ID and end the line with a period for consistency.

Reply:
Ah good catch - thanks Johnny! I'll add it whenever we deploy to Production next.